presto-orc/src/test/java/com/facebook/presto/orc/TestSelectiveOrcReader.java

```diff
@@ -1043,4 +1079,9 @@ private static String toCharValue(Object value, int minLength)
         }
         return values;
     }
+
+    private static List<Byte> toByteArray(List<Integer> ints)
+    {
+        return ints.stream().map((i) -> i == null ? null : i.byteValue()).collect(toList());
```

Comment:
toImmutableList

Reply:
can't use immutableList here as there are nulls in these arrays.